### ISSUE TITLE
Leviathan trading: "TypeError: button is undefined"

### ISSUE THREAD

**Issue by uvok** [cameroncondry/cbc-kitten-scientists#170]
With kittens game 1.1.8.6 / Kittens scientists version 1.3.0 (bookmarklet), I got the following in the console when the elders have arrived. Trading doesn't happen then:

```
12:01:55.859 TypeError: button is undefined
TradeManager.prototype.trade() kitten-scientists.user.js:787
Engine.prototype.trade() kitten-scientists.user.js:517
Engine.prototype.iterate() kitten-scientists.user.js:366
1 kitten-scientists.user.js:787:13
12:01:55.859 Array [ "Warning!", "unable to find trade button for " ]1 kitten-scientists.user.js:325:18

```

Strangely, this only appears sometimes. 

Could this have something to do with not being on the Trade tab? When they appear, after having been on the trading tab once, things seem to be fine.

I also have "Kitten extrapolation" installed as GreaseMonkey script, don't know if that influences it. 

Browser is Firefox 45.0.1 under Gentoo, 64bit.

**Comment by uvok** [comment]
While I was away and let the game run, Leviathans seem to have appeared. Console was full of these messages, trades did not happen.
So it only seems to "find" the button once you visit the Trade page, at least for me

**Comment by Saucistophe** [comment]
Duplicate of #206.

**Comment by Wymrite** [comment]
Fixed as of #248
